tests/Unit/Gutenberg/BlockInterfaceTest.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace WpOop\WordPress\Test\Unit\Gutenberg;
+
+use PHPUnit\Framework\TestCase;
+use WpOop\WordPress\Test\Stub\Gutenberg\Block;
+
+class BlockInterfaceTest extends TestCase
+{
+    public function testUsage()
+    {
+        $innerBlock = new Block('core/quote', [], [], '', []);
+        $subject = new Block(
+            'core/paragraph',
+            ['foo' => 'bar', 'foobar' => 42],
```

Comment:
We usually use randomized values where the exact value is not important. This ensures that the implementation is not rigged to pass the tests.

Reply:
You can rig even with random values, but why would we do that :) 

Here it seems to be fine, the main goal of these test is to check that the function signatures work as expected, etc., not to test logic.

Reply:
How would you rig an implementation if the values that are being tested for are random?

Reply:
Depends on what is tested, for example some kind of reflection hack, looking at backtrace, etc. :)

So like if you set a goal to rig tests, you probably can find a way.

Reply:
But anyway here adding Faker does not seem to have any benefits because

> the main goal of these tests is to check that the function signatures work as expected, etc., not to test logic.

Reply:
In this case, I'm not sure why you need to use any values in the test, or to call any of the methods.

Reply:
https://github.com/wp-oop/wordpress-interface/pull/4#discussion_r614076113

Reply:
Still don't understand why need to use the values. If you use them, and the implementation is not rigged, then use randomized values to prove that. If you don't need values because you are simply testing instantiation (signature compat etc), then please remove them entirely.